src/js/components/Drop/__tests__/Drop-test.js

```diff
@@ -15,6 +15,10 @@ const customTheme = {
   global: {
     drop: {
       shadowSize: 'large',
```

Comment:
can we deprecate `shadowSize` and use `elevation` more?

Reply:
Done.